frontend/src/services.js

```diff
@@ -623,6 +623,12 @@ CoreServices.factory('Pulse', ['$resource', '$cookies', function($resource, $coo
             params: { pulseId: '@pulseId' },
             headers: { 'X-CSRFToken': () => $cookies.csrftoken },
         },
+        test: {
+            url: '/api/pulse/test',
+            method: 'POST',
+            params: { pulseId: '@id' },
+            headers: { 'X-CSRFToken': () => $cookies.csrftoken },
```

Comment:
we can delete all that old CSRFToken stuff, that was specific to the django app.  i removed all of these from the other endpoints a little while ago.